src/zope/password/zpasswd.py

```diff
@@ -134,7 +139,14 @@ def __init__(self, options):
 
     def read_input_line(self, prompt):
         # The tests replace this to make sure the right things happen.
-        return raw_input(prompt)
+        # Py3 support
+        try:
+            answer = raw_input(prompt)
+        except (KeyboardInterrupt, EOFError):
+            print('\n')
+            print(os.path.basename(sys.argv[0]), 'was aborted.')
+            sys.exit(1)
+        return answer
```

Comment:
I don't understand why this is commented as `Py 3 support`.  raw_input() may raise either of those exceptions just as well on Python 2.